code/game/objects/items/melee/misc.dm

```diff
@@ -172,6 +172,7 @@
 	slot_flags = ITEM_SLOT_BELT
 	force = 12 //9 hit crit
 	w_class = WEIGHT_CLASS_NORMAL
+	custom_premium_price = 200
```

Comment:
This is too cheap for how easily it can be obtained, even in the sec gear vendor.